### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#202] [Esercizi - Bertini] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/406b9fb3-7807-4f43-a583-1a7c2dd760ec)

**Comment by cornflexxx** [comment, marked as answer]
## Soluzione analoga a quella in classe
Valore di attesa condizionato:


$$\mathbb{E}(X|y)=\sum_{x\in Im(X)} x\cdot\mathbb{P}(X=x|Y=y)=\sum_{x\in Im(X)} x\cdot\frac{\mathbb{P}(X=x,Y=y)}{\mathbb{P}(Y=y)}$$

--------------------------
$$\mathbb{E}(X)=\sum_{y\in Im(Y)}\mathbb{P}(Y=y)\sum_{x\in Im(X)} x\cdot\frac{\mathbb{P}(X=x,Y=y)}{\mathbb{P}(Y=y)}=\sum_{y\in Im(Y)}\sum_{x\in Im(X)} x\cdot\mathbb{P}(X=x,Y=y)=...$$

$$...=\sum_{x}x\sum_{y} \mathbb{P}(X=x,Y=y)=\qquad\text{Cambio l'ordine delle sommatorie, quindi porto fuori la x}$$


Osservo che $\sum_{y} \mathbb{P}(X=x,Y=y)$ rappresenta proprio la distribuzione marginale $\mathbb{P}(X=x)$, quindi : 

$$\sum_{x}x\sum_{y} \mathbb{P}(X=x,Y=y)=\sum_x x\cdot \mathbb{P}(X=x)$$

Che è  la definizione di valore atteso.
